### PR TITLE
Add user history support for auth and mgmt apis

### PR DESCRIPTION
**Pull request by dorsha**
## Description
Related to: https://github.com/descope/etc/issues/4713

Add user history support for auth and mgmt apis.

## Must
- [x] Tests

### CODE REVIEW

File: descope/api/client.go [hunk 1]
```diff
@@ -367,76 +372,99 @@ type mgmtEndpoints struct {
 func (e *endpoints) SignInOTP() string {
 	return path.Join(e.version, e.auth.signInOTP)
 }
+
```
Comment (aviadl) [inline descope/api/client.go:375]: Did you deliberately add all those spaces ?

**Review by aviadl — COMMENTED** [review]
Also, what about readme ?

File: descope/api/client.go (hunk 1 above)
Comment (dorsha) [reply descope/api/client.go:375]: yes

**Comment by dorsha** [comment]
> Also, what about readme ?

We don't have for Me so not sure if we need also For History. Since they are on the same place. 
But can be added if needed.

**Review by aviadl — DISMISSED** [review]

**Comment by dorsha** [comment]
Added readme.

**Review by aviadl — APPROVED** [review]

**Pull request merged**